backend/app/services/chat/url_parser.py

```
@@ -0,0 +1,740 @@
+
+            try:
+                # Navigate to the URL with timeout
+                # Use "domcontentloaded" instead of "networkidle" because some sites
+                # (like Weibo) never reach networkidle due to continuous resource loading
+                try:
+                    await page.goto(
+                        url,
+                        wait_until="domcontentloaded",
+                        timeout=self.browser_timeout * 1000,
+                    )
+                except Exception as goto_error:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)